indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -73,12 +73,27 @@
             $reviewBox.find('.js-edit-review').trigger('click');
             $('body, html').animate({scrollTop: $reviewBox.offset().top}, 'fast');
             $reviewBox.find('.i-timeline-item-box').effect('highlight', {color: Palette.highlight}, 'slow');
-        }).on('click', '.ratings-switch .i-button', function() {
+        }).on('click', '.js-ratings-toggle', function() {
             var $this = $(this);
+            $this.find('.js-show-ratings, .js-hide-ratings').toggleClass('weak-hidden');
+            var $reviewBox = $this.closest('.i-timeline-item-box');
+            if ($reviewBox.data('noComment') !== undefined) {
+                if ($reviewBox.hasClass('header-only')) {
+                    $reviewBox.toggleClass('header-only');
```

Comment:
Since you check for `hasClass` above, I'd rather have a `removeClass` here since it's essentially what happens.